### PR TITLE
Feature/issue #6 port rev1 page part views

### PR DESCRIPTION
**Pull request by Shemptankerous**
branch contains all my changes for page parts.  There are changes that came along from the event results stuff as well.

### CODE REVIEW

**Comment by johnnynibbles** [comment]
It looks like you inadvertantly moved src/DartLeague/DartLeague.Web/Controllers into src/DartLeague/DartLeague.Web/Configurations.

You will need to move those back to where they were before we can merge this in.

**Comment by Shemptankerous** [comment]
I moved the controllers folder back out of the configurations folder and pushed.

**Comment by johnnynibbles** [comment]
I included a couple of changes to formatting around the home page and the result from page part message.  Removed the h2 as a default around the text from the page part.

File: src/DartLeague/DartLeague.Web/ViewComponents/NavigationViewComponent.cs [hunk 1]
```diff
@@ -226,17 +226,17 @@ await BuildOtherMenu()
                         Title = "Site",
                         Href = "#",
                         SubNavigations = {
-                            new NavigationViewModel{ Title = "Dart Events", Href = url.Action("Index", "DartEvent", new {Area = "Site" })},
-                            new NavigationViewModel{ Title = "Page Content", Href =url.Action("Index","PagePart",new {Area = "Manage"})}
+                            new NavigationViewModel{ Title = "Dart Events", Href = url.Action("Index", "DartEvent", new { Area = "Site" }) },
+                            new NavigationViewModel{ Title = "Page Content", Href =url.Action("Index", "PagePart", new { Area = "Site" }) }
```
Comment (johnnynibbles) [inline src/DartLeague/DartLeague.Web/ViewComponents/NavigationViewComponent.cs]: I also fixed the navigation after the move from manage to site

Comment (Shemptankerous) [inline src/DartLeague/DartLeague.Web/ViewComponents/NavigationViewComponent.cs]: Is this in good shape to merge then?

**Pull request merged**